### PR TITLE
Update article.md

### PR DESCRIPTION
**Pull request by gleachkr**
Fixed a tiny typo.

### CODE REVIEW

**Comment by joaquinelio** [comment]
👍

**Comment by gleachkr** [comment]
Yeah, "this convention" might be a little stuffy. What about a slightly bigger edit:

> But following this pattern helps us replace `Map` with `Set` in certain cases with ease, and vice versa.

**Comment by joaquinelio** [comment]
It's ok anyway, it doesn't need to be perfect.

mm...
if we overthink about it we can lose the original idea (it may sound dubious on purpose, like making excuses). so I prefer the first fix.

But now I am overthinking.

**Comment by joaquinelio** [comment]
I'm not native English, still learning.
Without the fix, it sounds like a suggestion; am I right?

**Comment by gleachkr** [comment]
Hm... it doesn't quite sound like a suggestion to me. It's just saying that the callback having three arguments is a design choice that could have been made differently, but it has good reasons behind it. 

But if putting it this way makes it hard to understand, then again, maybe that's not the best formulation. Just dropping the original edit to 

> That's for compatibility with `Map` where the callback passed `forEach` has three arguments. Looks a bit strange, for sure. But this may help to replace `Map` with `Set` in certain cases with ease, and vice versa.

 would be OK too.

There's also maybe another tiny grammar nitpick for this paragraph. I think "the callback passed `forEach`" should be "the callback passed to `forEach`". I can do a separate PR or fold it into this one, if you agree with the edit.

**Comment by joaquinelio** [comment]
I like the added "this" alone, 
`But this may help to replace`


sorry for
"Without the fix, it sounds like a suggestion; am I right?"
nothing to do with the repo, 
tldr
it was my interpretation of the original, not what the author intended to say.
`But may help to replace (...)` 
sounded to me like
`But replacing (...) may help`
I wasnt sure if was bad grammar or just a wrong way to put it.

**Comment by gleachkr** [comment]
> `But may help to replace (...)`
> sounded to me like
> `But replacing (...) may help`
> I wasnt sure if was bad grammar or just a wrong way to put it.

Oh, I see! Yeah, to me, without the fix, the original just feels like a typo. It's definitely ungrammatical as is.

**Comment by joaquinelio** [comment]
I'm not the one to say 
sorry, maybe the first fix was ok

"passed to" is needed too I think

And thanks 👍 for the explanation.
